.github/workflows/test.yaml

```diff
@@ -137,6 +137,15 @@ jobs:
           bundler-cache: true
       - run: bundle install
       - run: bundle exec rbs collection install
+      - run: bundle exec rbs-inline --output lib/
+      - name: Check for uncommitted changes
+        run: |
+          if [ -n "$(git status --porcelain)" ]; then
+            echo "Uncommitted changes detected after running rbs-inline."
+            echo "Please run 'bundle rake rbs_inline' and commit the changes generated by rbs-inline." >&2
+            exit 1
+          fi
+        shell: bash
```

Comment:
If there is an uncommitted sig/generated/lrama type definition, it fails as follows:

![error](https://github.com/user-attachments/assets/ab920959-737e-4949-8713-6872ceb29b29)